jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormation.kt

```diff
@@ -42,13 +42,15 @@ fun CloudFormationClient.describeStack(stackName: String, callback: (Stack?) ->
     }
 }
 
-fun CloudFormationClient.describeStackForSync(stackName: String, callback: (Stack?) -> Unit) {
+fun CloudFormationClient.describeStackForSync(stackName: String, enableParamsAndTags: (Boolean) -> Unit, callback: (Stack?) -> Unit) {
     ApplicationManager.getApplication().executeOnPooledThread {
         try {
+            enableParamsAndTags(false)
             val stack = this.describeStacks { it.stackName(stackName) }.stacks().firstOrNull()
+            enableParamsAndTags(true)
             callback(stack)
         } catch (e: Exception) {
-            /* no-op */
+            enableParamsAndTags(true)
```

Comment:
only need to write `enableParamsAndTags(true)` once
```kt
try {
...
} catch (_: ValidationError) {
    // noop because we don't need to surface "stack not found" to the user
} finally {
    enableParamsAndTags(true)
}
```

Reply:
Yes, I was thinking of doing that, but the location of enableParams andTags in the try block is before the other UI changs, hence implemented in both try-catch block